packages/xrpl/src/models/utils/flags.ts

```diff
@@ -117,7 +120,15 @@ export function convertTxFlagsToNumber(tx: Transaction): number {
     }, 0)
   }
 
-  return 0
+  return Object.keys(tx.Flags).reduce((resultFlags, flag) => {
```

Comment:
If the previous `if` condition is True (`isTxToFlagKey(tx.TransactionType)`), then the Global flags are not parsed. i.e. In cases where transaction-specific flags are specified, this return statement is not executed.

Reply:
Good catch - fixed